.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,26 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Run PyTest
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  run_lint:
+    uses: lsst/rubin_workflows/.github/workflows/lint.yaml@main
```

Comment:
I'll remake the comment  from #29 that these lines should not be here because there is already a lint.yaml workflow in this repo -- it's just that that repo needs to be modernized to the current standard of a reusable workflow (see my comment on the other PR).